packages/trace-viewer/src/ui/networkTab.tsx

```diff
@@ -87,7 +91,8 @@ export const NetworkTab: React.FunctionComponent<{
     onHighlighted={item => onEntryHovered(item?.resource)}
     columns={visibleColumns(!!selectedEntry, renderedEntries)}
     columnTitle={columnTitle}
-    columnWidth={columnWidth}
+    columnWidths={widths}
```

Comment:
I would instead use key={} to split grids with different column sets. key can be a join of column names.